packages/react-ui-core/src/Ratings/RatingItem.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+export default class RatingItem extends Component {
+  static propTypes = {
+    theme: PropTypes.object,
+    fillWidth: PropTypes.string,
+    index: PropTypes.number,
+    backgroundColor: PropTypes.string,
+  }
+  static defaultProps = {
+    theme: {},
+  }
+
+  render() {
+    const {
+      fillWidth,
+      index,
+      backgroundColor,
+      theme,
+      pathD,
+      polygonPoints,
+    } = this.props
+    const id = `rItem_${index}`
```

Comment:
This is falsely assuming that this component will only rendered a single time (per index) on a given page, when it will in all likelihood get rendered multiple times, resulting in numerous duplicate ID's.